test/prettier.mjs

```
@@ -25,11 +25,14 @@ import eslintPluginPug from 'eslint-plugin-pug';
 import vueEslintParser from 'vue-eslint-parser';
 import * as eslintPluginGraphql from '@graphql-eslint/eslint-plugin';
 import eslintMdx from 'eslint-mdx';
+import eslintPluginJson from '@eslint/json';
 
 const rule = eslintPluginPrettier.rules.prettier;
 const RuleTester =
   eslintUnsupportedApi.FlatRuleTester ?? eslintPackage.RuleTester;
-const ESLint = eslintUnsupportedApi.FlatESLint ?? eslintPackage.ESLint;
+const ESLint = eslintUnsupportedApi.FlatESLint ?? eslint.ESLint;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)